cmplibrary/src/main/java/com/sourcepoint/gdpr_cmplibrary/GDPRConsentLib.java

```diff
@@ -354,32 +385,58 @@ public void onFailure(ConsentLibException exception) {
         });
     }
 
-    void showView(View view) {
+    void showView(View view, boolean isFromPM) {
```

Comment:
@Suryakantj , pls take this second argument out of this method, as i have comment on this PR already.